frontend/apps/filemanager/filemanagerfilesearcher.lua

```diff
@@ -74,28 +70,22 @@ function FileSearcher:setSearchResults()
 end
 
 function FileSearcher:close()
-    if self.search_value then
         UIManager:close(self.search_dialog)
-        if string.len(self.search_value) > 0 then
-            self:readDir() --- @todo this probably doesn't need to be repeated once it's been done
-            self:setSearchResults() --- @todo doesn't have to be repeated if the search term is the same
-            if #self.results > 0 then
-                self:showSearchResults() --- @todo something about no results
-            else
-                UIManager:show(
-                    InfoMessage:new{
-                        text = BaseUtil.template(_("No results for '%1'"),
-                                             self.search_value)
-                    }
-                )
+        self:readDir() --- @todo this probably doesn't need to be repeated once it's been done
+        self:setSearchResults() --- @todo doesn't have to be repeated if the search term is the same
+        if #self.results > 0 then
+            self:showSearchResults() --- @todo something about no results
```

Comment:
Seems to be my fault, in part anyway. https://github.com/koreader/koreader/commit/92f0092ad4ee7de35014a41f75e2f5b4f9e550d3
```suggestion
            self:showSearchResults()
```